dimagi/utils/couch/migration.py

```diff
@@ -120,16 +134,23 @@ def _migration_get_couch_model_class(cls):
         """
         raise NotImplementedError()
 
-    def _migration_get_or_create_couch_object(self):
+    def _migration_get_couch_object(self):
         cls = self._migration_get_couch_model_class()
         if not self.couch_id:
+            return None
+        else:
+            obj = cls.get(self.couch_id)
```

Comment:
no need to block on this, but it would be slightly cleaner to just return from this line